cli/command/prune/prune.go

```diff
@@ -37,7 +37,25 @@ func RunContainerPrune(dockerCli command.Cli, filter opts.FilterOpt) (uint64, st
 
 // RunVolumePrune executes a prune command for volumes
 func RunVolumePrune(dockerCli command.Cli, filter opts.FilterOpt) (uint64, string, error) {
-	return volume.RunPrune(dockerCli, filter)
+	// Volume prune does not support the "until" filter. Unfortunately
+	// opts.FilterOpt cannot easily be cloned, nor is it possible
+	// to get a list of all filters that were set.
+	//
+	// To "remove" the "until" filter if set, we just copy the only supported
+	// filter for now ("label").
+	//
+	// Keep this hack in sync with supported filters for volume prune to prevent
+	// new filters from silently being ignored!
+	//
+	// TODO make `opts.FilterOpt` more flexible to prevent awkward hacks like this
+	// TODO version this hack once "until" filter is supported for volumes
+	vf := opts.NewFilterOpt()
+	if filter.Value().Include("label") {
```

Comment:
If anyone has suggestions for doing this in a cleaner way, let me know

I didn't want to update the package upstream (moby/moby) to keep the diff small